speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/racemanmenu.cpp

```diff
@@ -38,16 +38,15 @@
 #include <drivers.h>
 #include <cars.h>
 
-#include <playerconfig.h>
-#ifdef CLIENT_SERVER
-#include <csnetwork.h>
-#else
-#include <network.h>
-#endif
+#include <controlconfig.h>
+
 
 #include "legacymenu.h"
 #include "racescreens.h"
 
+#include "mainmenu.h"
+
+// SIMULATED DRIVING ASSISTANCE: removed networking
```

Comment:
no need for a comment for the #includes. Just putting it in the CHANGES.txt is enough